src/RootApp.tsx

```diff
@@ -18,7 +18,9 @@ const RootApp = () => (
                 </WebConfigProvider>
             </UserSettingsProvider>
         </ApiProvider>
-        <ReactQueryDevtools initialIsOpen={false} />
+        {window.Proxy && ( // '@tanstack/query-devtools' requires 'Proxy', which cannot be polyfilled for legacy browsers
```

Comment:
Should we also check `browser.tv`?

Reply:
To hide it on TVs?

Reply:
Yes since it makes navigation weird... although I have been thinking about adding a custom menu entry or something for it because I hate the floating button 😅